### DISCUSSION TITLE
v1.6.2 - Further Stability improvement, Ventura fixes & minor features

### DISCUSSION THREAD

**Discussion by nodeful** [bitgapp/eqMac#727] [General]
#### Features & Improvments:
* Volume Mute indicator & ability to toggle.
* Driver Installation/Update is now performed from the app, no more PKG Installer, using DMG instead.
* Added ability for user to decide to show or hide the Driver from the device list
* You can now Reset Output device Cache in case you see duplicate devices
* App Volume Mixer - Ability to Show Inactive apps, Reorder and Filter apps you don't want to control
* Using High DPI Status Bar Icon as well as App Icons in App Volume Mixer

#### Bug Fixes:
* Improved way of tracking AirPods In-Ear Play/Pause functionality.
* Fixed audio delays introduced in v1.5.x
* Fixed more crashes related to many factors
* When user switches macOS to an unsupported device, eqMac now switches off and shows error message instead of crashing
* Improved Output device caching logic to avoid duplication
* Ventura Beta issues with no audio or slowed down audio fixed

<hr /><em>This discussion was created from the release <a href='https://github.com/bitgapp/eqMac/releases/tag/v1.6.2'>v1.6.2 - Further Stability improvement, Ventura fixes & minor features</a>.</em>

**Comment by silasmahler** [comment]
Unfortunately after update the audio driver install fails

**Reply by nodeful** [reply]
Hey, where have you places the eqMac.app, in which directory?
Do you get any errors telling you what went wrong?

**Reply by silasmahler** [reply]
Its in the programms folder and execution permissions are given
